apple-news.php

```diff
@@ -51,9 +51,9 @@ function apple_news_date( $format, $timestamp = null, $timezone = null ) {
  */
 function apple_news_activate_wp_plugin() {
 	// Check for PHP version.
-	if ( version_compare( PHP_VERSION, '5.3.6' ) < 0 ) {
+	if ( version_compare( PHP_VERSION, '8.0.0' ) < 0 ) {
 		deactivate_plugins( basename( __FILE__ ) );
-		wp_die( esc_html__( 'This plugin requires at least PHP 5.3.6', 'apple-news' ) );
+		wp_die( esc_html__( 'This plugin requires at least PHP 8.0.0', 'apple-news' ) );
```

Comment:
I think this is oudated. And it doesn't reflect what we are advertising to support.

Reply:
Good catch!